src/app/conf/2025/code-of-conduct/opengraph-image.tsx

```diff
@@ -0,0 +1,10 @@
+import { SimpleOpengraphImage } from "../components/og-images/simple-opengraph-image"
+export {
+  generateStaticParams,
+  contentType,
+  size,
+} from "../components/og-images/simple-opengraph-image"
+
+export default SimpleOpengraphImage.bind(null, {
```

Comment:
We're using `bind` here instead of an arrow or a new function because `@types/react` doesn't seem to know we can return `Promise<JSX>` just yet.

This should be solved after migrating to React 19, but I'm fine with taking this small stylistic/readability hit just to a void dealing with React versions for now.